### PR TITLE
Landlord and application changes

### PR DESCRIPTION
**Pull request by shivamchawla29june**

### CODE REVIEW

**Review by ssreepathy — COMMENTED** [review]
Pushed changes with landlord view and applications

**Review by ssreepathy — APPROVED** [review]
Merging application and landlord view

File: src/App.js [hunk 1]
```diff
@@ -1,40 +1,39 @@
...
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header> */}
+        {/* <div>Here</div> */}
```
Comment (jessicamecht) [inline src/App.js:27]: Can we remove the comment?

File: src/App.js [hunk 2]
```diff
@@ -1,40 +1,39 @@
...
+        <Routes>
+          <Route exact path="/" element={<Listings />} />
+          <Route path="/listingInfo" element={<ListingInfo />} />
+          <Route path="/listingSearch" element={<ListingSearch />} />
+          <Route path="/application" element={<ApplicationInfo />} />
+        </Routes>
       </div>
```
Comment (jessicamecht) [inline src/App.js:34]: Is the signup view still connected?

File: src/containers/landlord/listing.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import "./listing.scss";
+import {useNavigate} from "react-router-dom";
+
+
+const Listing = (props) => {
+  //console.log(props.name);
+  const data = {
```
Comment (jessicamecht) [inline src/containers/landlord/listing.jsx:8]: can probably remove comment

File: src/containers/landlord/listing.scss [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+  align-items: center;
+  padding: 20px;
+  margin: 20px;
+  // gap: 11px;
+
+  // position: absolute;
+  width: 591px;
```
Comment (jessicamecht) [inline src/containers/landlord/listing.scss:11]: remove comment here as well?

**Review by jessicamecht — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (shivamchawla29june) [reply src/App.js:27]: We will need to replace this comment with Navbar, hence kept the comment. Will remove in the next PR

**Pull request merged**